### DISCUSSION TITLE
Add "Duplicate" item to tab's context menu

### DISCUSSION THREAD

**Discussion by vasso-etc** [sumatrapdfreader/sumatrapdf#3773] [Ideas]
Action like "Open in new window" but open same file in current window (on the tab to the right of the current one).

**Comment by GitHubRulesOK** [comment]
discussed a long time ago that to add every command like `CTRL+SHIFT+N` makes the dropdown list so much longer so I add my own like this (in theory you could add OPEN in new window there but not ideal)  OR
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/578e7462-2d01-4066-bde4-0b7ddd388360)
**an alternative is to use your own hotkey so for example just pressing N**
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2f7e9b7a-258e-42cf-8720-00b324ab8a96)

**Comment by gaoqiangks** [comment]
Is there any plan for this feature?

**Reply by GitHubRulesOK** [reply]
That function is currently covered by "open in new window" (you can assign a key or use the tab)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/adcaec05-f6f0-44db-a34d-0677b314db29)
The way that happens is a new tabbed window opens you can drag to either side OR drag and drop tab back to main window
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/0a15ed58-b95a-4a3a-9a54-401f47396f21)

Current disadvantage (was an open issue I think some time ago) is the new window is very much close to existing window just slightly offset, but slightly better than the "directly on top" method Edge uses for similar ability)

**Reply by gaoqiangks** [reply]
Thanks. It's more pleasant if there's an option such that I an reopen the current file in a new tab, without further operations.

**Reply by GitHubRulesOK** [reply]
I understand but the simplest is drag that new tab to the underneath yellow icon, takes a little practice
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/fdc6ec04-2e5b-4b9a-bc79-46686d049373)

@kjk can the offset be increased ? downwards would make it easier to drag new tab upwards.
